internal/services/codesigning/registration.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package codesigning
+
+import (
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+)
+
+type Registration struct{}
+
+var _ sdk.TypedServiceRegistration = Registration{}
+
+func (r Registration) AssociatedGitHubLabel() string {
+	return "service/codesigning"
```

Comment:
We should probably just call this 
```suggestion
	return "service/trustedsigning"
```